### PR TITLE
Create Oracles.md

### PR DESCRIPTION
**Pull request by JKtranslator**

### CODE REVIEW

File: docs/infrastructure/Oracles.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+
+### Deployed contracts
+
+The main oracle contracts on Group 0 are deployed at the following addresses: 
+
+
+
+* **Testnet**: vKdWgyqtQZzAV7eoMXxQCkyiNtceViGZheXC1iHbKFDZ
+* **Mainnet:** 285zrkZTPpUCpjKg9E3z238VmpUBQEAbESGsJT6yX7Rod
```
Comment (nop33) [inline docs/infrastructure/Oracles.md]: Would recommend these changes:

```suggestion
- **Testnet**: `vKdWgyqtQZzAV7eoMXxQCkyiNtceViGZheXC1iHbKFDZ`
- **Mainnet:** `285zrkZTPpUCpjKg9E3z238VmpUBQEAbESGsJT6yX7Rod`

File: docs/infrastructure/Oracles.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+Each asset is updated every two minutes if the new price deviates from the old one by more than 0.5%. Additionally, a heartbeat of 1 hour is applied, which means that each price is updated at least once per day even if it moves by less than 0.5%.
+
+
+## Example
+
+You can find a deployment example [here](https://github.com/alephium/ralph-example/tree/master/call-oracle/contracts).
+
+## Support
+
+For developer assistance, connect with the DIA team directly on[ Discord](https://chat.openai.com/c/d2ca764b-00d6-4de7-8555-0cc30ac416b1#) or [Telegram](https://t.me/diadata_org).
```
Comment (nop33) [inline docs/infrastructure/Oracles.md]:
```suggestion
For developer assistance, connect with the DIA team directly on [Discord](https://chat.openai.com/c/d2ca764b-00d6-4de7-8555-0cc30ac416b1#) or [Telegram](https://t.me/diadata_org).
```

File: docs/infrastructure/Oracles.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
+---
+sidebar_position: 30
+sidebar_label: Oracles
+title: Oracles
+---
+
+# Ecosystem Price Oracles, powered by DIA
+
+
```
Comment (nop33) [inline docs/infrastructure/Oracles.md]: In Markdown, 2 (or more) empty lines are the same as 1. This document includes many unnecessary extra empty lines which might cause issues with translators. I'd recommend removing them, although it's not a biggy

File: docs/infrastructure/Oracles.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+1. The price of the asset, with 8 decimals.
+2. The timestamp of the last update in Unix time format, in UTC timezone.
+
+
+### Included price feeds
+
+The Alephium oracle includes the following price feeds. Use the Query Symbol as a parameter in the `get-value` function to receive the latest price information for any of these assets.
+
+
+<table>
```
Comment (nop33) [inline docs/infrastructure/Oracles.md]: Markdown supports tables: https://www.markdownguide.org/extended-syntax/#tables

File: docs/infrastructure/Oracles.md (hunk 1 above)
Comment (JKtranslator) [reply docs/infrastructure/Oracles.md]: fixed

File: docs/infrastructure/Oracles.md (hunk 3 above)
Comment (JKtranslator) [reply docs/infrastructure/Oracles.md]: fixed

File: docs/infrastructure/Oracles.md (hunk 2 above)
Comment (JKtranslator) [reply docs/infrastructure/Oracles.md]: fixed

File: docs/infrastructure/Oracles.md (hunk 4 above)
Comment (JKtranslator) [reply docs/infrastructure/Oracles.md]: changed

**Pull request merged**